ch03tests/05Mocks.ipynb

```diff
@@ -4,54 +4,75 @@
    "cell_type": "markdown",
```

Comment:
<p>This text should be placed before the above call, so that we are telling the reader what's comign.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UCL/rsd-engineeringcourse/pull/221/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>